src/test/java/com/commercetools/sync/products/helpers/VariantReferenceResolverTest.java

```diff
@@ -497,67 +519,72 @@ void isProductReference_WithProductReferenceAttribute_ShouldReturnTrue() {
     }
 
     @Test
-    void getResolvedIdFromKeyInReference_WithEmptyValue_ShouldGetEmptyId() {
+    void getProductResolvedIdFromKeyInReference_WithEmptyValue_ShouldGetEmptyId() {
```

Comment:
What do you mean by empty Id 🤔 could it be better with ShouldReturnEmptyString ?

Reply:
792ab66